providers/org/plos/normalizer.py

```diff
@@ -14,12 +15,48 @@ class Contributor(Parser):
     order_cited = ctx('index')
 
 
+class Link(Parser):
+    url = ctx
+    type = Static('doi')
+
+
+class ThroughLinks(Parser):
+    link = Delegate(Link, ctx)
+
+
+class Publisher(Parser):
+    name = ctx
+
+
+class Association(Parser):
+    entity = Delegate(Publisher, ctx)
+
+
 class Publication(Parser):
+
+    def format_doi_as_url(self, doi):
+        return format_doi_as_url(self, doi)
+
     title = XPath(ctx, "str[@name='title_display']").str['#text']
     description = XPath(ctx, "arr[@name='abstract']/str").str
     contributors = Map(
         Delegate(Contributor),
         XPath(ctx, "arr[@name='author_display']").arr.str
     )
+    publishers = Map(
+        Delegate(Association),
+        XPath(ctx, "str[@name='journal']").str['#text']
+    )
+    links = Map(
+        Delegate(ThroughLinks),
+        RunPython(
+            'format_doi_as_url',
+            XPath(ctx, "str[@name='id']").str['#text']
+        )
+    )
     date_published = ParseDate(XPath(ctx, "date[@name='publication_date']").date['#text'])
-    # doi = ctx.xpath("str[@name='id']").str['#text']
+
+    class Extra:
+        eissn = XPath(ctx, "str[@name='eissn']").str['#text']
```

Comment:
I think this should be a type of link. Eissns don't have a way of being expressed as UR[I|L|N]s currently, which makes this a bit odd.
Let's go with `eissn://<value>` and type as `esissn`